apps/dashboard/app/(app)/@breadcrumb/authorization/permissions/[permissionId]/page.tsx

```diff
@@ -0,0 +1,83 @@
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  BreadcrumbLink,
+  BreadcrumbList,
+  BreadcrumbPage,
+  BreadcrumbSeparator,
+} from "@/components/ui/breadcrumb";
+
+import { BreadcrumbSkeleton } from "@/components/dashboard/breadcrumb-skeleton";
+import { getTenantId } from "@/lib/auth";
+import { db } from "@/lib/db";
+import { Suspense } from "react";
+
+export const dynamic = "force-dynamic";
+export const runtime = "edge";
+
+type PageProps = {
+  params: { permissionId: string };
+};
+
+async function AsyncPageBreadcrumb(props: PageProps) {
+  const tenantId = getTenantId();
+
+  const workspace = await db.query.workspaces.findFirst({
```

Comment:
this query is super expensive and not cached, why do you need all of those relations?

Reply:
all you need is the name from a permission, no?

Reply:
that's true, I should remove that

Reply:
yeah, do it like you did for the ratelimit namespaces, that query is good